tests/test_corona.py

```diff
@@ -278,19 +278,21 @@ def test_pad_cumulative_series_to_yesterday():
 
 def test_germany_get_population():
     germany = c.germany_get_population()
-    assert germany > 8200000
-    hamburg = c.germany_get_population("Hamburg")
+    assert germany.shape == (412, 38)
+    hamburg = germany.loc['SK Hamburg'].population
     assert hamburg > 1800000
-    pinneberg = c.germany_get_population(landkreis="LK Pinneberg")
+    pinneberg = germany.loc['LK Pinneberg'].population
     assert pinneberg > 30000
 
 
 def test_get_population():
-    germany = c.get_population("Germany")
-    assert germany > 8200000
-    bayern = c.get_population("Germany", "Bayern")
-    assert bayern > 10000000
-    russia = c.get_population("Russia")
-    assert russia > 140000000
-    kyoto = c.get_population("Japan", "Kyoto")
-    assert kyoto > 2500000
+    world = c.get_population()
+    assert world.shape == (188, 6)
```

Comment:
same here: at some point they may decide to remove those unlucky ships or add some more ships. Or status of some place changes (say Taiwan or Hong-Kong or Crimea or whatever else).

Reply:
Same as my reply above, made the tests more specific, now it checks that the index name is correct, that population is a column, and that the countries in the population table match up with the countries we have cases for.